luarules/gadgets/unit_collision_damage_behavior.lua

```diff
@@ -139,14 +152,13 @@ function gadget:GameFrame(frame)
 				velX = (velocityCap / velocityLength) * velX
 				velY = (velocityCap / velocityLength) * velY
 				velZ = (velocityCap / velocityLength) * velZ
-			end
-			if velocityLength > objectCollisionVelocityThreshold then
-				local decelerateHorizontal = 0.95
+			elseif velocityLength > velocitySlowdownThreshold then
+				local decelerateHorizontal = 0.75
```

Comment:
This is a functional change, is that intended?
```lua
if A then
  foo()
elseif B then
  bar()
end
```
If A is true, the elseif block will not be run, regardless of the value of B

Reply:
Yes. If A then velocity is sheared off to a reasonable value. This is from the initial explosion if impulse is too great.

Elseif B the value is decreased incrementally in a smoother fashion to make the deceleration look less janky.